docs/content/glossary/glossary.mdx

```diff
@@ -0,0 +1,318 @@
+---
+title: Glossary
+---
+
+# Flow Glossary
+
+## All words and definitions related to Blockchain and Flow
+
+## **A**
+
+## Authentication
+[Authentication](https://docs.onflow.org/fcl/reference/authentication/#gatsby-focus-wrapper) connects users to their wallets, 
+and the wallet provides the account information and services through FCL.
+
+## Authorization
+Authorization is a process by which a server determines if the client has permission to use a resource or access a file. In FCL, 
+An authorization service knows the Account and the Key to sign the transaction when the service is sent to FCL during authentication.
```

Comment:
this needs to be simplified. Unclear what it means that a service is sent to FCL. I'd say authorization is the mechanism that verifies that a prompted user action is in fact intended by the user. The user has to authorize an action that will create/update a record on the blockchain. FCL helps with authorization by facilitating the transaction signing process

Reply:
fixed